padinfo/core/button_info.py

```diff
@@ -92,36 +90,43 @@ def _get_sub_attr_multiplier(self, monster_model):
             return 1 / 10
         return 1 / 3
 
-    def to_string(self, monster, info):
-        card_btn_str = self._get_card_btn_damage(CARD_BUTTONS, info, monster)
-        team_btn_str = self._get_team_btn_damage(TEAM_BUTTONS, info, monster)
-        return INFO_STRING.format(monster.monster_id, monster.name_en, info.main_damage, info.sub_damage,
-                                  info.total_damage,
-                                  info.main_damage_with_atk_latent, info.sub_damage_with_atk_latent,
-                                  info.total_damage_with_atk_latent, card_btn_str, team_btn_str)
+    def to_string(self, monster, info, dgcog):
```

Comment:
can you make dgcog the first argument here? just a convention we try to follow I guess

you SHOULD be able to do this by right-clicking in pycharm, refactor, edit signature & it'll auto-update all the usages